REVISED_README.md

```diff
@@ -0,0 +1,72 @@
+<kbd>![image](https://user-images.githubusercontent.com/92825997/184505554-d5617a88-df44-4bec-9c1e-eddbc2673e88.png)</kbd><br><br>
+Then, create a new file by clicking `Add file` and then `Create new file`<br><br>
+<kbd>![image](https://user-images.githubusercontent.com/92825997/184505565-2f6afcd2-a4e9-4b13-868e-82b3b9ab6634.png)</kbd><br>
+<kbd>![image](https://user-images.githubusercontent.com/92825997/184505579-9e701c4b-72c4-4e2d-8f5a-0226fad1c466.png)</kbd><br><br>
+Then, name the new file as your Github username such as `win21H2`<br>
+
+[↑ Go to top ↑](#quick-access-index)
+
+### Step 2 - Inputting information in the JSON file
+In the file you just made, copy and paste the below template:<br>
+```json
+{
+  "name": "",
+  "url": "",
+  "image": "",
+  "why": "",
+  "weekly": [],
+  "goal": "",
+  "location": ""
+}
+```
+<br>
+and fill out the information (make sure to look at the notes below)<br><br>
+
+ - Name: You can use your github name, actual name, or a pseudonym!
+ - Url: Please insert Github link here, we will expand the user json files to include socials in the future
+ - Image: You can use any appropriate profile image here
+ - Why : Please write down your reason why you have joined the #FitDevs community!
+ - Weekly: Your goals for fitness, can be food/exercise/mental/emotional, whatever fitness means to YOU
+ > Such as the below
+```json
+["Running 6x", "Abs 6x", "Pushups 6x"]
+["Drinking 500ml Water 3/day", "Meatless Mondays 2x/month", "Sleeping 7h 6/week"]
+```
+ - Goal: What is your overarching goal?
+ - Location: Be as vague or as specific as you would like to
+ > A full example below:<br><br>
+ <kbd>![image](https://user-images.githubusercontent.com/92825997/184505823-f83b2f5b-1547-45a0-b460-c120c1173495.png)</kbd><br>
+
+[↑ Go to top ↑](#quick-access-index)
+
+### Step 3 - Commit the file
+Make sure that before you click the `Propose new file` button, that you have made the below changes:
+ - Fill out the title, this can just be `Create JSON file
+ - Fill out a summary, this can just be `created my JSON file in the data directory`
+ - Create a new branch for this commit and start a pull request
+<br><br>
+<kbd>![image](https://user-images.githubusercontent.com/92825997/184505853-a843023c-d168-4e97-9f9d-28c9b9d39939.png)</kbd><br>
+
+[↑ Go to top ↑](#quick-access-index)
+
+### Step 4 - Create a pull request
+By now, you should be brought to a new page, click the green `Create pull request` button and if everything works, it should look like the last image. Congrats! you have just created your own "card" which will be shown on the <a href="https://fitdevs-withkat.github.io/Fitness-Accountability/">main website</a>
+<br><br>
+<kbd>![image](https://user-images.githubusercontent.com/92825997/184505866-6dbed0bd-4925-4ba1-8132-d4af4def4dfe.png)</kbd><br>
+<kbd>![image](https://user-images.githubusercontent.com/92825997/184505881-e7280a6c-4192-465f-a5c4-4114c40438f6.png)</kbd><br>
+
```

Comment:
Maybe we could add a bit about what to expect after they've posted their PR. We can just add a note, something like 'your PR will be manually reviewed by a member of the GitHub team and merged.'